mail.php

```diff
@@ -20,88 +20,172 @@
 
 declare(strict_types=1);
 
+namespace WordPlate;
+
+if (!defined('ABSPATH')) {
+    exit(); // Exit if accessed directly
+}
+
 // If the environment function doesn't exist, we don't want to continue.
 if (!function_exists('env')) {
     return;
 }
 
-// Add custom SMTP credentials.
-add_action('phpmailer_init', function (PHPMailer $mail) {
-    $mail->IsSMTP();
-    $mail->SMTPAuth = env('MAIL_USERNAME') && env('MAIL_PASSWORD');
-
-    $mail->SMTPAutoTLS = false;
-    $mail->SMTPSecure = env('MAIL_ENCRYPTION', 'tls');
+/**
+ * This is the mail class.
+ *
+ * @author Daniel Gerdgren <daniel@gerdgren.se>
+ * @author Vincent Klaiber <hello@vinkla.com>
+ */
+final class Mail
```

Comment:
Lets move this class to `src/Mail.php` and autoload it using `psr-4` in `composer.json`.

Reply:
We should also add the namespace `WordPlate\Mail` to the file.